frontend/src/views/Join.tsx

```diff
@@ -155,6 +156,9 @@ function JoinGamePage() {
             // Input is disabled if loading or if no nickname exists, has a space, or is too long.
             disabled={!validRoomId || loading}
           />
+          <TextLink to="/game/create">
+            Or create a room &#8594;
+          </TextLink>
```

Comment:
Out of curiosity, what made you add this?

Reply:
The reason was cause whenever you get redirected (kicked, exited game, etc.) you land on the join page, and sometimes users might want to create a room instead. The only way to currently do that is to click the top-left logo back to the home page and then create a room. Feel free to lmk if you prefer any alternative solutions to this tho (copied from notes)